packages/ui/src/components/molecules/OnboardingLayout.tsx

```diff
@@ -8,29 +8,55 @@ import {
   DialogBoxBackgroundHeader,
   DialogBoxBackgroundHeaderProps,
 } from './DialogBoxBackground';
+import { InfoAside, InfoAsideProps } from './Aside/InfoAside';
 
-export interface OnboardingLayoutProps extends AsideProps {
+export interface OnboardingLayoutProps
+  extends Partial<AsideProps>,
+    Partial<InfoAsideProps> {
   children: ReactNode | undefined;
   headerProps?: DialogBoxBackgroundHeaderProps;
   footerProps?: DialogBoxBackgroundFooterProps;
 }
 
+const aside: FC<Partial<AsideProps> & Partial<InfoAsideProps>> = ({
+  text,
+  title,
+  subTitle,
+  version,
+  img,
+  currentState,
+  totalState,
+}) => {
+  if (title && subTitle && version) {
+    return <InfoAside title={title} subTitle={subTitle} version={version} />;
+  }
+  if (img && text && currentState && totalState) {
+    return (
+      <Aside
+        img={img}
+        text={text}
+        currentState={currentState}
+        totalState={totalState}
+      />
+    );
+  }
```

Comment:
If there is `currentState` and `totalState`, use `Aside` otherwise use `InfoAside`.

You might want to make things optional in both aside components to make it work.